apps/judicial-system/web/src/routes/Shared/Cases/cases.graphql

```diff
@@ -94,6 +94,7 @@ query Cases {
     indictmentVerdictAppealDeadline
     indictmentRulingDecision
     indictmentDecision
+    indictmentCompletedDate
```

Comment:
I think this is causing an error in schema generation. Is this here by accident or did you mean to add this to the case query as well?

Reply:
Yeees true actually, I did mean to add it in there but I forgot to add it in to the model in the API layer, fixing now